### PR TITLE
[KYUUBI #2478][FOLLOWUP] Fix bin/beeline without -u exits unexpectedly

### LINKED ISSUE #3030: [Bug] `bin/beeline` without `-u` exits unexpectedly
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

```
bin/beeline
Warn: Not find kyuubi environment file /Users/kentyao/incubator-kyuubi/dist/conf/kyuubi-env.sh, using default ones...
org.apache.hive.beeline.BeeLine.defaultBeelineConnect()
```

the command above exits with no reason whereas `bin/beeline -u 'jdbc:hive2://localhost:10009/'` works properly

### Affects Version(s)

master

### Kyuubi Server Log Output

```logtalk
not related.
```

### Kyuubi Engine Log Output

```logtalk
not related.
```

### Kyuubi Server Configurations

_No response_

### Kyuubi Engine Configurations

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### _Why are the changes needed?_

fix #3030 

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
image

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-hive-beeline/pom.xml [hunk 1]
```diff
@@ -136,6 +136,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (turboFei) [inline kyuubi-hive-beeline/pom.xml:143]: move the test dependency to the end of dependencies

**Review by turboFei — APPROVED** [review]
left minor comments

**Comment by turboFei** [comment]
thanks, merged to master

**Comment by turboFei** [comment]
also backport to branch 1.5
